topics/mvvmcross/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: mvvmcross-plugin, xamarin-mvvmcross
+created_by: Stuart Lodge
+display_name: MvvmCross
+github_url: https://github.com/mvvmcross
+logo: mvvmcross.png
+related: xamarin, dotnet, framework
+released: June 2013
+short_description: The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows, and Mac.
+topic: mvvmcross
+url: https://www.mvvmcross.com/
+---
+MvvmCross is a cross-platform model-view-view model framework that enables developers to create powerful cross platform mobile applications. It supports Xamarin.iOS, Xamarin.Android, Xamarin.Mac, Xamarin.Forms, Universal Windows Platform (UWP), and Windows Presentation Framework (WPF).
```

Comment:
The "cross platform" in "powerful cross platform mobile" should be hyphenated as a compound adjective. 👀 You could also add a comma after "powerful" if you wanted.